lib/game-resolver.js

```diff
@@ -9,7 +11,13 @@ export const lookup = (type) => {
     }
   }
 
-  const game = games[type]
+  let game = games[type]
+  
+  Object.keys(games).forEach((id) => {
+    if (games[id].extra && games[id].extra.old_id && !options.skipOldIDs) {
+      game = games[id]
+    }
+  })
```

Comment:
This does the `skipOldIDs` check on every game, which is not ideal.
You should wrap this iteration in that `skipOldIDs` check (so it checks just one time).
Also, instead of `games[id].extra && games[id].extra.old_id` I think you could do `games[id]?.extra.old_id`

Reply:
Sorry, but you mean moving the `Object.keys(games).forEach()` iteration to `bin/gamedig.js`?

Reply:
Maybe I haven't explained it that well
```js
if (!options.skipOldIDs) {
  Object.keys(games).forEach((id) => {
    if (games[id]?.extra.old_id) {
      game = games[id]
    }
  })
}
```

Reply:
Oh, yes, of course! haha Thank you! 😅